src/plotting.jl

```diff
@@ -52,24 +54,35 @@ function plot_graph(g::SimpleWeightedGraph, locs_x=nothing, locs_y=nothing;
         edgestrokec=edge_colors)#, kwargs...)
 end
 
+function plot_graph(g::SimpleWeightedGraph, coords::Coordinates, path=nothing, kwargs...)
```

Comment:
If I understand Julia correctly, `kwargs` should be in the keyword section to disallow calls like
```julia
plot_graph(graph, coords, path=nothing, my_unnamed_arg_1, my_unnamed_arg_2, my_unnamed_arg_3)
```
Also, it looks like `path` should be in the keyword argument section too.
```suggestion
function plot_graph(g::SimpleWeightedGraph, coords::Coordinates; path=nothing, kwargs...)
```